src/CodeStyle/CSharp/Tests/Properties/launchSettings.json

```diff
@@ -1,12 +1,14 @@
 {
   "profiles": {
-    "xUnit.net Console": {
-      "executablePath": "$(XUnitPath)",
-      "commandLineArgs": "$(XUnitArguments)"
+    "xUnit.net Console (32-bit)": {
+      "commandName": "Executable",
+      "executablePath": "$(NuGetPackageRoot)xunit.runner.console\\$(XUnitVersion)\\tools\\net452\\xunit.console.x86.exe",
+      "commandLineArgs": "$(TargetPath) -noshadow -verbose"
     },
-    "xUnit.net WPF Runner": {
-      "executablePath": "$(XUnitWpfPath)",
-      "commandLineArgs": "$(XUnitWpfArguments)"
+    "xUnit.net Console (64-bit)": {
+      "commandName": "Executable",
+      "executablePath": "$(NuGetPackageRoot)xunit.runner.console\\$(XUnitVersion)\\tools\\net452\\xunit.console.exe",
+      "commandLineArgs": "$(TargetPath) -noshadow -verbose"
     }
   }
-}
+}
```

Comment:
❔  Why did all of these break? Can they be fixed?

Reply:
@sharwell Sorry, not sure what you mean?

Reply:
![image](https://user-images.githubusercontent.com/1408396/52883771-ae5d0100-3131-11e9-83da-e14d983bde9b.png)

It's saying look how it's now wrong

Reply:
Oh you mean the newline? I just didn't know what you were referring to. :smile:

I can spend time to change it. Is it worth it? I personally don't care about this debate. Why it change is probably either because I used vim at one point which sometimes changes this, or the .json VS formatter did it.